### ISSUE TITLE
IconButtons wrapped by muted tooltips do not respect aria-label set on button

### ISSUE THREAD

**Issue by csongnguyen** [Workday/canvas-kit#1036]
## 🐛 Bug Report

With a muted tooltip wrapping the IconButton, the aria-label is undefined and no aria-label appears on the button element.

## To Reproduce

Without tooltip:
```
<IconButton
            aria-label="Activity Stream"
            icon={activityStreamIcon}
            size={IconButton.Size.Medium}
            variant={IconButton.Variant.InverseFilled}
          />
```
renders
```
<button aria-label="Activity Stream" > ... </button>
```

With tooltip:
```
    <Tooltip title={"some title"} type="muted">
        <IconButton
          aria-label="Activity Stream"
          variant={IconButton.Variant.Plain}
          size={IconButton.Size.Medium}
          icon={icon}
          onClick={onClick}
        />
    </Tooltip>
```
renders
```
<button> ... </button>
```

## Expected Behavior

An IconButton should have an aria-label attribute set when wrapped in a muted tooltip.

## Actual Results

An IconButton wrapped in a muted tooltip does not have an aria-label attribute.

## Browser (if applicable)

If this was a visual bug, what browser did you spot it on?

## Link to repl or repo (highly encouraged)

Please provide a reference to a repo or branch to help us reproduce this bug.

### Error Output

Paste the error output here

```bash

```

or

Include a screenshot of a visual bug
